controller/work-item-children-blackbox_test.go

```diff
@@ -0,0 +1,248 @@
+	require.NotNil(s.T(), workItemLinkCategory)
+	s.userLinkCategoryID = *workItemLinkCategory.Data.ID
+	s.T().Logf("Created link category with ID: %s\n", *workItemLinkCategory.Data.ID)
+
+	// Create work item link type payload
+	createLinkTypePayload := createParentChildWorkItemLinkType("test-bug-blocker", workitem.SystemBug, workitem.SystemBug, s.userLinkCategoryID, s.userSpaceID)
+	_, workItemLinkType := test.CreateWorkItemLinkTypeCreated(s.T(), s.svc.Context, s.svc, s.workItemLinkTypeCtrl, createLinkTypePayload)
+	require.NotNil(s.T(), workItemLinkType)
+	s.bugBlockerLinkTypeID = *workItemLinkType.Data.ID
+	s.T().Logf("Created link type with ID: %s\n", *workItemLinkType.Data.ID)
+}
+
+// The TearDownTest method will be run after every test in the suite.
+func (s *workItemChildSuite) TearDownTest() {
+	s.clean()
+}
+
+//-----------------------------------------------------------------------------
+// helper method
+//-----------------------------------------------------------------------------
+
+// createParentChildWorkItemLinkType defines a work item link type
+func createParentChildWorkItemLinkType(name string, sourceTypeID, targetTypeID, categoryID, spaceID uuid.UUID) *app.CreateWorkItemLinkTypePayload {
+	description := "Specify that one bug blocks another one."
+	lt := link.WorkItemLinkType{
+		Name:           name,
+		Description:    &description,
+		SourceTypeID:   sourceTypeID,
+		TargetTypeID:   targetTypeID,
+		Topology:       link.TopologyTree,
+		ForwardName:    "parent of",
+		ReverseName:    "child of",
+		LinkCategoryID: categoryID,
+		SpaceID:        spaceID,
+	}
+	reqLong := &goa.RequestData{
+		Request: &http.Request{Host: "api.service.domain.org"},
+	}
+	payload := link.ConvertLinkTypeFromModel(reqLong, lt)
+	// The create payload is required during creation. Simply copy data over.
+	return &app.CreateWorkItemLinkTypePayload{
+		Data: payload.Data,
+	}
+}
+
+//-----------------------------------------------------------------------------
+// Actual tests
+//-----------------------------------------------------------------------------
+
+// In order for 'go test' to run this suite, we need to create
+// a normal test function and pass our suite to suite.Run
+func TestSuiteWorkItemChildren(t *testing.T) {
+	pwd, err := os.Getwd()
+	if err != nil {
+		require.Nil(t, err)
+	}
+	resource.Require(t, resource.Database)
+	suite.Run(t, &workItemChildSuite{DBTestSuite: gormtestsupport.NewDBTestSuite(pwd + "/../config.yaml")})
+}
+
+func (s *workItemChildSuite) TestListChildren() {
+	createPayload := CreateWorkItemLink(s.bug1ID, s.bug2ID, s.bugBlockerLinkTypeID)
+	_, workItemLink := test.CreateWorkItemLinkCreated(s.T(), s.svc.Context, s.svc, s.workItemLinkCtrl, createPayload)
+	require.NotNil(s.T(), workItemLink)
+
+	createPayload2 := CreateWorkItemLink(s.bug1ID, s.bug3ID, s.bugBlockerLinkTypeID)
```

Comment:
You might as well move the creation of the two links into `SetupTest`. Then you only need to store the parent bug ID (`bug1ID`) in the suite, because you're comparing the bug's titles and not their IDs below. Which means, that you can shrink these variables:

```go
	bug1ID               uint64
 	bug2ID               uint64
 	bug3ID               uint64
 	userLinkCategoryID   uuid.UUID
 	bugBlockerLinkTypeID uuid.UUID
 	userSpaceID          uuid.UUID
```

to just 

```go
	bug1ID               uint64
```